### ISSUE TITLE
[Consistency] User encounters a component or pattern that isn't considered a best practice.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80114]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefits Non-Disability
**Product:** Burial Form 530
**Feature:** 21P-530 Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** User encounters a component or pattern that isn't considered a best practice.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @it-harrison (QA)

### Description

The breadcrumbs are not rendered by using the Design System `va-breadcrumbs` web component

## Recommended action

Use the Design System <va-breadcrumbs> web component

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Breadcrumbs ](https://design.va.gov/components/breadcrumbs)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by sortizsh** [comment]
Not sure if we can affect this or if it's a platform level change

**Comment by shiragoodman** [comment]
@sortizsh the ticket is stating that your product is not using the `va-breadcrumbs` web component. I'm not sure I follow how this would be a platform level change.

**Comment by steele-lm** [comment]
Noting that there is other work to align the breadcrumbs (and url) with design standards in #76700

**Comment by mjknight50** [comment]
Hi @shiragoodman - Our team has been working on a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/31318) to address this ticket and to use the new va-breadcrumbs component.  However, it doesn't seem like breadcrumbs component is fully baked yet. 

From @it-harrison 
As far as using breadcrumbs in content-build there is an upcoming issue that DST will be working on to deal with that, see [here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2945).

Once that PR is done, we will circle back to this ticket.

For more context:
https://dsva.slack.com/archives/C057W3N9K2S/p1723569704515969

**Comment by it-harrison** [comment]
@mjknight50 - that sounds good, DST should pick be picking it up soon

**Comment by steele-lm** [comment]
Marking this as blocked until the DST update is released.

**Comment by steele-lm** [comment]
@matt4su I believe @DLarson-Oddball has a PR ready for this work, but it cannot be released until the Design System Team completes the related work.

**Comment by steele-lm** [comment]
Design system PR that will allow this ticket to move forward: https://github.com/department-of-veterans-affairs/content-build/pull/2254#issuecomment-2393885220

**Comment by steele-lm** [comment]
Design system PR was released. Closing this ticket.
